package/MDAnalysis/core/universe.py

```diff
@@ -1436,6 +1439,19 @@ def from_smiles(cls, smiles, sanitize=True, addHs=True,
 
         return cls(mol, **kwargs)
 
+    def guess_topoloyAttribute(self, context, to_guess):
```

Comment:
```suggestion
    def guess_TopologyAttr(self, context, to_guess):
```

There's already a method called `add_TopologyAttr`, and the class is called a `TopologyAttr`. Going with the [principal of least astonishment](https://en.wikipedia.org/wiki/Principle_of_least_astonishment) that advises a consistent API, could you please rename this method to something that users could easily guess (😛) themselves?